submissions/description/sensitive_data_exposure/disclosure_of_secrets/non_corporate_user/template.md

```diff
@@ -0,0 +1,22 @@
+# Disclosure of Secrets for Non-Corporate User
+
+## Overview of the Vulnerability
+
+Disclosure of secrets occurs when sensitive data is not behind an authorization barrier. When this information is exposed it can place the application at further risk of compromise. This application discloses secrets for a non-corporate user which can be leveraged by an attacker to access the application and make requests on the legitimate user’s behalf.
```

Comment:
1 technical wording change

> Disclosure of secrets occurs when the data is properly secured, When this information is exposed it can place the application at further risk of compromise. This application discloses secrets for a non-corporate user which can be leveraged by an attacker to access the application and make requests on the legitimate user’s behalf.